src/hooks/useTxNotifications.ts

```diff
@@ -16,23 +16,20 @@ import useSafeAddress from './useSafeAddress'
 import { getExplorerLink } from '@/utils/gateway'
 
 const TxNotifications = {
-  [TxEvent.SIGN_FAILED]: 'Signature failed. Please try again.',
-  [TxEvent.PROPOSED]: 'Your transaction was successfully proposed.',
-  [TxEvent.PROPOSE_FAILED]: 'Failed proposing the transaction. Please try again.',
-  [TxEvent.SIGNATURE_PROPOSED]: 'You successfully signed the transaction.',
-  [TxEvent.SIGNATURE_PROPOSE_FAILED]: 'Failed to send the signature. Please try again.',
-  [TxEvent.EXECUTING]: 'Please confirm the execution in your wallet.',
-  [TxEvent.PROCESSING]: 'Your transaction is being processed.',
-  [TxEvent.PROCESSING_MODULE]:
-    'Your transaction has been submitted and will appear in the interface only after it has been successfully processed and indexed.',
-  [TxEvent.ONCHAIN_SIGNATURE_REQUESTED]:
-    'An on-chain signature is required. Please confirm the transaction in your wallet.',
-  [TxEvent.ONCHAIN_SIGNATURE_SUCCESS]:
-    "The on-chain signature request was confirmed. Once it's on chain, the transaction will be signed.",
-  [TxEvent.PROCESSED]: 'Your transaction was successfully processed and is now being indexed.',
-  [TxEvent.REVERTED]: 'Transaction reverted. Please check your gas settings.',
-  [TxEvent.SUCCESS]: 'Your transaction was successfully executed.',
-  [TxEvent.FAILED]: 'Your transaction was unsuccessful.',
+  [TxEvent.SIGN_FAILED]: 'Failed to sign. Please try again.',
+  [TxEvent.PROPOSED]: 'Successfully added to queue.',
+  [TxEvent.PROPOSE_FAILED]: 'Failed to add to queue. Please try again.',
+  [TxEvent.SIGNATURE_PROPOSED]: 'Successfully signed.',
+  [TxEvent.SIGNATURE_PROPOSE_FAILED]: 'Failed to send signature. Please try again.',
+  [TxEvent.EXECUTING]: 'Confirm the execution in your wallet.',
+  [TxEvent.PROCESSING]: 'Validating...',
+  [TxEvent.PROCESSING_MODULE]: 'Validating module interaction...',
+  [TxEvent.ONCHAIN_SIGNATURE_REQUESTED]: 'Confirm on-chain signature in your wallet.',
+  [TxEvent.ONCHAIN_SIGNATURE_SUCCESS]: 'On-chain signature request confirmed.',
+  [TxEvent.PROCESSED]: 'Successfully validated. Indexing...',
+  [TxEvent.REVERTED]: 'Reverted. Please check your gas settings.',
+  [TxEvent.SUCCESS]: 'Successfully executed.',
+  [TxEvent.FAILED]: 'Failed.',
```

Comment:
Why are these changes necessary? If there's no `humanDescription`, we now lose some context.

Reply:
`humanDescription` will be always there unless the feature is disabled for technical reasons.

Reply:
If the signature hash does not match on the gateway the `humanDescription` will be `null`. You can test this with spending limits atm:

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/4fd7a4eb-18bd-4f24-985b-0c6965ebc959)

Reply:
Good point, I think we should add a fallback for the title. Maybe we can include the nonce, something like `Transaction #113`